docs/edge.rst

```diff
@@ -318,19 +318,177 @@ the form.
+
+
+Set up DNS
+''''''''''
+
+Here you set up the DNS:
+
+.. code-block:: console
+
+    68.183.5.88 ub12
+    178.62.239.7 ub11
+    188.166.9.90 ub10
+
+
+Set up cluster
+''''''''''''''
+
+Next is the command for the three-node cluster:
+
+.. code-block:: console
+
+    microk8s add-node
+
+
+Join nodes to cluster
+'''''''''''''''''''''
+
+Now you join the nodes to the cluster:
+
+.. code-block:: console
+
+    root@ub11:~# microk8s join 188.166.9.90:25000/<cluster id>
+    Contacting cluster at 188.166.9.90
+    Waiting for this node to finish joining the cluster...
+
+
+Use a storage solution
+''''''''''''''''''''''
+
+The Microk8s setup will require a storage solution. In this case, the tutorial
+shows how to do so using `Longhorn`_, a distributed storage solution for
+Kubernetes. You can follow the `Longhorn installation instructions`_ as
+described below. (Other storage solutions for Kubernetes may work as well.)
+
+First the installation:
+
+.. code-block:: console
+
+    kubectl apply -f https://raw.githubusercontent.com/longhorn/longhorn/v1.1.1/deploy/longhorn.yaml
+
+Then you need to specify the root directory:
+
+.. code-block:: console
+
+    k8 -n longhorn-system edit deployment longhorn-driver-deployer
+
+    - name: KUBELET_ROOT_DIR
+    value: /var/snap/microk8s/common/var/lib/kubelet
+
+
+Set up Cloud region
+'''''''''''''''''''
+
+At this stage, you can create an Edge region via the CrateDB Cloud Console.
+Follow the steps outlined above :ref:`from the CrateDB sign up onwards
+<edge-signup>` to proceed.
+
+
+Run the script
+''''''''''''''
+
+Run the script with the following command:
+
+.. code-block:: console
+
+    wget -qO- https://console.cratedb.cloud/edge/cratedb-cloud-edge.sh > edge-installer.sh
+    chmod u+x edge-installer.sh
+    ./edge-installer --dry-run  <token>
+
+Note that ``dry-run`` provides, as the name suggests, a method to test the
+installation by running the YAML files that are going to be applied. This gives
+you an opportunity to verify them before the full install.
+
+The ``<token>`` in question is the token you receive from the CrateDB Console
+Edge region field in the Regions tab of the Organization Overview. For more
+information on this section of the CrateDB Console, refer to our :ref:`CrateDB
+Cloud Console overview <cloud-reference:overview-org-regions>`.
+
+With this, you should be ready to use CrateDB Edge via Microk8s.
+
+
+Microk8s Dashboard
+''''''''''''''''''
+
+With the Microk8s setup, you can optionally use the Microk8s Dashboard. To
+install this dashboard, run the following in the CLI:
+
+.. code-block:: console
+
+    microk8s enable dashboard
+    microk8s kubectl edit -n kube-system service/kubernetes-dashboard (change PORT to 8443)
+    ssh -i ~/.ssh/id_ed25519 root@<IP address> -L 10443:localhost:10443
+
+    token=$(microk8s kubectl -n kube-system get secret | grep default-token | cut -d " " -f1)
+    microk8s kubectl -n kube-system describe secret $token
```

Comment:
These instructions didn't actually work for me... I can see the dashboard is there, but no clue how to access it.